docs/mosviz/plugins.rst

```diff
@@ -47,10 +47,9 @@ Redshift Slider
 ---------------
 
 .. warning::
-    Using the redshift slider with many active spectral lines can be slow, as
-    every line gets replotted at each slider position. We recommended using 
-    the slider with no more than around a dozen lines plotted. You can deselect
-    lines using, e.g., the "Hide All" button in the line lists UI.
+    Using the redshift slider with many active spectral lines cause performance issues.
+    If reactions to the redshift slider are laggy, try plotting less lines. You can deselect
```

Comment:
Your suggestion is definitely more concise... but do you think it's still clear enough that the lag is referring to the movement of the spectral lines, and not the slider itself?  If that is still clear, I'll adopt your suggestion.  Otherwise maybe something like: "If the shifting of spectral lines lag behind the slider, try plotting less lines"?

Reply:
Yeah, I like "If the shifting of spectral lines lag behind the slider, try plotting less lines." Thanks!